docs/src/installation_upgrade.md

```diff
@@ -236,6 +236,179 @@ When versions are not directly upgradable, the old version needs to be
 removed before installing the new one. This won't affect user data but
 only the operator itself.
 
+### Upgrading to 1.21 from a previous minor version
+
+!!! Important
+    If you are upgrading from 1.19 or older, please read the
+    [instructions to upgrade to version 1.20](#upgrading-to-120-from-a-previous-minor-version)
+    first.
+
+With the goal to keep improving out-of-the-box the *convention over
+configuration* behavior of the operator, CloudNativePG 1.21 changes the default
+value of several knobs in the following areas:
+
+- startup and shutdown control of the PostgreSQL instance
+- self-healing
+- security
+- labels
+
+Most of the changes will affect new PostgreSQL clusters only.
+
+!!! Warning
+    Please read carefully the list of changes below, and how to change the
+    `Cluster` manifests to retain the existing behavior if you don't want to
+    disrupt your existing workloads. Alternatively, postpone the upgrade to 1.21
+    until you are sure. In general, we recommend adopting these default
+    values unless you have valid reasons not to.
+
+If you want to keep the existing behavior of CloudNativePG explicitly,
+(we advise not indefinitely,) you need to set these values in all your `Cluster`
+definitions **before upgrading** to version 1.21:
+
+```yaml
+spec:
+   ...
+   startDelay: 30
+   stopDelay: 30
+   switchoverDelay: 40000000
+   enableSuperuserAccess: true
+   replicationSlots:
+     highAvailability:
+       enabled: false
+```
+
+Once the upgrade is completed, also add:
+
+```yaml
+spec:
+   ...
+   smartShutdownTimeout: 30
+```
+
```

Comment:
```suggestion
The `smartShutdownTimeout` field has been introduced in the version
1.21, hence the need to add it after upgrading the operator.
```

Reply:
It will also be available in versions 1.20.3 and 1.19.5, so we should conform this to the others places where we mention this.